Library/Formula/libmpdec.rb

```diff
@@ -0,0 +1,13 @@
+class Libmpdec < Formula
+  desc "mpdecimal is a package for correctly-rounded arbitrary precision decimal floating point arithmetic"
```

Comment:
This description can be simplified to "Library for correctly-rounded arbitrary precision decimal floating point arithmetic."